.github/workflows/ci.yml

```
@@ -192,14 +192,20 @@ jobs:
       - uses: actions/checkout@v4
       - name: build
         run: |
-          cmake -B build -G "Visual Studio 17 2022" -A ${{matrix.arch}}
+          cmake -B build -D BUILD_EXAMPLES=ON -G "Visual Studio 17 2022" -A ${{matrix.arch}}
```

For consistency, write as:
```suggestion
          cmake -B build -DBUILD_EXAMPLES=ON -G "Visual Studio 17 2022" -A ${{matrix.arch}}
```